package.json

```diff
@@ -30,21 +34,35 @@
     "@nestjs/platform-express": "^8.4.4",
     "@nestjs/swagger": "^5.2.1",
     "@nestjs/throttler": "^2.0.1",
+    "@openapitools/openapi-generator-cli": "^2.5.2",
+    "@snapshot-labs/snapshot.js": "^0.3.87",
     "@types/passport": "^1.0.7",
     "@types/passport-http": "^0.3.9",
+    "apollo-cache-inmemory": "^1.6.6",
+    "apollo-client": "^2.6.10",
+    "apollo-link-http": "^1.5.17",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
     "cron": "^1.8.2",
+    "ethers": "^5.6.6",
     "eventsource": "^1.1.0",
+    "graphql": "^16.5.0",
+    "graphql-tag": "^2.12.6",
     "helmet": "^5.0.2",
+    "jest-openapi": "^0.14.2",
     "joi": "^17.6.0",
+    "keccak": "^3.0.2",
+    "lodash": "^4.17.21",
     "mongoose": "^6.2.4",
+    "node-fetch": "2.6.1",
     "openapi-types": "^10.0.0",
     "passport": "^0.5.2",
     "passport-headerapikey": "^1.2.2",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^7.2.0",
+    "siwe": "^2.0.3",
+    "snapshot-strategies": "file:../snapshot-strategies",
```

Comment:
file?

Reply:
just a hack to borrow a version of their strategies. Should be removed in the future. but since we're not using it, I could remove the entire snapshot module already, what do you think @wpmonty ?

Reply:
Up to you; just want to be sure if we deploy or other devs use that it is useable